### PR TITLE
add DataDog agent

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

**Comment by vladikoff** [comment]
@jrgm @dannycoates r?

I invited you both to the fxa-testing-dog account, so you should be able to see this URL: https://app.datadoghq.com/dash/host/12919524?live=true

File: roles/datadog/handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+
+- name: restart datadog
+  action: service name=datadog-agent state=restarted
```
Comment (jrgm) [inline roles/datadog/handlers/main.yml]: Does this need to be `sudo: true`. I got these errors trying to run this branch.

NOTIFIED: [datadog | restart datadog] ***************************************** 
failed: [ec2-52-24-229-252.us-west-2.compute.amazonaws.com] => {"failed": true}
msg: rm: cannot remove ‘/var/run/datadog-supervisord.pid’: Permission denied
rm: cannot remove ‘/var/lock/subsys/datadog-agent’: Permission denied
Password: su: Authentication failure
rm: cannot remove ‘/var/run/datadog-supervisord.pid’: Permission denied
rm: cannot remove ‘/var/lock/subsys/datadog-agent’: Permission denied

File: roles/datadog/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+datadog_api_key: "4f3b967bbf13c7769ac4fa89efda0fae"
+
+# Comma seperated list of tags
+datadog_tags: ""
```
Comment (jrgm) [inline roles/datadog/defaults/main.yml]: Is this intended to be the empty string (or now, I suppose)?

Comment (vladikoff) [inline roles/datadog/defaults/main.yml]: these are **global** datadog tags. No use for them yet

File: roles/datadog/handlers/main.yml (hunk 1 above)
Comment (vladikoff) [inline roles/datadog/handlers/main.yml]: added more sudo.

**Comment by vladikoff** [comment]
Updated.

**Comment by jrgm** [comment]
LGTM. Merging...

**Comment by vladikoff** [comment]
Thank you!!

**Pull request merged**